README.md

```diff
@@ -1,73 +1,61 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
```

Comment:
Please add the project title here. Kindly do remember that README should have complete information about the project.

Reply:
Thank you. Let me add it